test/integration/test_integration_rioxarray.py

```diff
@@ -540,6 +540,64 @@ def test_clip_geojson__no_drop(open_func, invert, from_disk, expected_sum):
         assert clipped_ds.test_data.rio.nodata == xdi.rio.nodata
 
 
+def dummy_dataset_non_geospatial():
```

Comment:
It might make sense to convert this to a pytest fixture.

Reply:
I'm using also the same geometries as in `test_clip_geojson`, so this could also be condensed by a fixture?

Reply:
How would i go about to make the constant visible for the functions without a fixture? Or just leave both in their functions?

Reply:
Justin is talking about the geometries only in this scenario. So, you would change `def geometries ...` into a global variable at the top of the file like `_CLIP_GEOMETRIES=[...]`

Reply:
Ok, i see. So should this be changed to a global variable, or reverted to the geometries in the functions?

Reply:
Either way works for me :+1: